subread/2.0.0/FeatureCounts.nf

```diff
@@ -11,11 +11,13 @@ process FeatureCounts {
     file(genome_gtf)   
   
     output:
-    tuple file("${run_id}_*_featureCounts.txt"), file("${run_id}_${params.fc_count_type}_featureCounts_matrix.txt"), file("${run_id}_biotype_featureCounts_matrix.txt"), file("${run_id}_*_featureCounts.txt.summary")
+    path "${run_id}_${params.fc_count_type}_featureCounts.raw.txt"
+    path "${run_id}_${params.fc_count_type}_featureCounts.txt.summary"
+    path "${run_id}_biotype_featureCounts.{matrix.txt,txt.summary}", optional: true
```

Comment:
Why did you change 'file' to 'path'?

Reply:
I guess it's fine if we can merge it to master once approved. I was under the impression that there is an exception for hotfixes to be immediately released since they are usually quite critical. Let's discuss our release cycle then.

Indeed, I should fix the module names. I switched to using the path syntax since this should be prefered over file according to the manual for nextflow versions above 19.0.0. Secondly,  I could not get named/optional outputs to work with the file(..) syntax.

Reply:
Yes it is common to release hotfixes immediately, but then you should create a pull request to 'master', not 'develop'. We can't merge develop to master as a hotfix because it is possible that develop contains other changes as well.

A I see: `The path qualifier should be preferred over file to handle process input files when using Nextflow 19.10.0 or later.` Did not know that, we should probably fix that in more modules.

Thus, code is good, but pull request should be from dev-ubec (or a hotfix branch) to master.